src/Sparrow/Compression/LZ4.cs

```diff
@@ -793,24 +793,26 @@ private static TResult ThrowException<TResult>(Exception e)
         }
 
         [MethodImpl(MethodImplOptions.AggressiveInlining)]
-        private static void WildCopy(byte* destPtr, byte* srcPtr, byte* destEndPtr)
+        private static void WildCopy(byte* dest, byte* src, byte* destEnd)
         {
-            // JIT: We make local copies of the parameters because the JIT will not be able to figure out yet that it can safely inline
-            //      the method cloning the parameters. As the arguments are modified the JIT will not be able to inline it.
-            //      This wont be needed anymore when https://github.com/dotnet/coreclr/issues/6014 is resolved.
+            do
+            {
+                ((ulong*)dest)[0] = ((ulong*)src)[0];
+                if (dest + sizeof(ulong) >= destEnd)
+                    return;
 
-            byte* dest = destPtr;
-            byte* src = srcPtr;
-            byte* destEnd = destEndPtr;
+                ((ulong*)dest)[1] = ((ulong*)src)[1];
```

Comment:
Problem is, this is not a copy... It is a repeater (mascareding as copy). Changing the behavior implies trying to find if we have overlaps and that incurr cost we cannot afford to pay (we went -2x trying to do that).